README.md

```diff
@@ -1,8 +1,14 @@
 # 461_1_23
-Mimi Chon
-Anna Shen
-Emile Baez
-Ben Schwartz
+Team Members:
+Mimi Chon, Anna Shen, Emile Baez, Ben Schwartz
+
+# Building program
+In command line, first run 
+<go build -o "run">
+in order to create executable with 'run' as name.
+
+# CLI
+<build, install, test, 'URL_FILE'> commands are recognized where 'URL_FILE' must be an absolute path to a file in the system.
```

Comment:
Could we add a "help" command to list this information? Even though it's not specified in the requirements, I think it's pretty common in CLIs. If this is more work than just a quick fix, we can add it to our backlog instead.

Reply:
We could add help as a command. There is a -h (help) flag for each command already. As long as we don't get points docked off for it I'm down to include it.